qir/qsharp-target-tests/TargetTestScripts/Intrinsic/ApplyControlledZ.qs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace QATSample {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Measurement;
+
+    @EntryPoint()
+    operation Main(): Result
+    {
+      use control = Qubit();
+      use target = Qubit();
+
+      ApplyControlledZ(control, target);
```

Comment:
Same here, this is an internal function that is not expected to be called outside of the target package implementation.